src/components/TextField/TextField.tsx

```diff
@@ -152,3 +63,7 @@ export default class TextField extends React.PureComponent<TextFieldProps, {}> {
     return classes.join(' ');
```

Comment:
same thing here, we could simplify this with our join utility:

```
className={join(['y-base-text-field' 'y-text-field', className, prefix, suffix])}
```

Reply:
I updated the TextArea with your suggestion.  This one isn't quite as simple.  `prefix` and `suffix` don't contain the class names.  I took another look at the implementation, and I'm not seeing much room to simplify without sacrificing readability.  Let me know if you have another suggestion though.